test/e2e-common.sh

```diff
@@ -372,3 +374,14 @@ function wait_for_file() {
   done
   return 0
 }
+
+function install_cert_manager(){
+  url="$(curl -s https://api.github.com/repos/cert-manager/cert-manager/releases/latest | jq '.assets[] | select(.name|match("cert-manager.yaml$")) | .browser_download_url')"
+
+  kubectl apply \
+    -f "${url}" || return 1
+  
+  scale_controlplane cert-manager cert-manager-cainjector cert-manager-webhook
+
+  wait_until_pods_running ${SYSTEM_NAMESPACE} || fail_test "Failed to install cert manager"
```

Comment:
We need to `wait_until_pods_running ${SYSTEM_NAMESPACE}` on the cert-manager namespace and not the system namespace